.github/workflows/reusable/ops-summarize-failure.yml

```
@@ -0,0 +1,53 @@
+        description: Workflow run ID to summarize.
+        type: string
+        required: true
+    secrets:
+      anthropic-api-key:
+        description: Anthropic API key.
+        required: true
+      github-token:
+        description: GitHub token with actions:read.
+        required: true
+
+outputs:
```

actionlint (1.7.12)

[error] unexpected key "outputs" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)